backend/src/filter/filter.service.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@nestjs/common';
+import { Neo4jService } from 'nest-neo4j/dist';
+import { FilterServiceBase } from './filter.service.base';
+import {
+  EdgeTypeFilterModel,
+  FilterModelEntry,
+  NodeTypeFilterModel,
+} from '../shared/filter';
+
+@Injectable()
+export class FilterService implements FilterServiceBase {
+  constructor(private readonly neo4jService: Neo4jService) {}
+
+  public async getNodeTypeFilterModel(
```

Comment:
comment missing

Reply:
I opened #141 for this and will implement this in a follow up.